configure

```diff
@@ -571,13 +571,7 @@ get_system_exe_extension() {
 
 # Show the configure help line for an option
 option_help() {
-	if test "${3}" != "" ; then
-		tmpopt_prefix="${3}"
-	else
-		tmpopt_prefix="--"
-	fi
-	tmpopt=`echo $1 | sed 's/_/-/g'`
-	option=`echo "${tmpopt_prefix}${tmpopt}                       " | sed "s/\(.\{23\}\).*/\1/"`
+	option=$(echo "${3:---}${1}                       " | sed -e "s/\(.\{23\}\).*/\1/" -e "s/_/-/g")
```

Comment:
Not that this may substitute underscores by dashed in prefix

Reply:
Yes. In practice, we never put any underscore when we set a `${3}` value, though. By the way, `${3}` is always set to the same value: two spaces.

So yeah, it's a small behavior change but I don't think it will cause any issue.